docs/standard/serialization/system-text-json-how-to.md

```diff
@@ -189,12 +189,16 @@ To deserialize from a file by using asynchronous code, call the <xref:System.Tex
 > [!TIP]
 > If you have JSON that you want to deserialize, and you don't have the class to deserialize it into, you have options other than manually creating the class that you need:
 >
-> 1. Use [JsonDocument and Utf8JsonReader](system-text-json-use-dom-utf8jsonreader-utf8jsonwriter.md) directly.
-> 1. Use Visual Studio 2019 to automatically generate the class you need:
->    1. Copy the JSON that you need to deserialize.
->    1. Create a class file and delete the template code.
->    1. Choose **Edit** > **Paste Special** > **Paste JSON as Classes**.
->    The result is a class that you can use for your deserialization target.
+> * Deserialize into a [JSON DOM (document object model)](system-text-json-use-dom-utf8jsonreader-utf8jsonwriter.md) and extract what you need from the DOM.
+>
+>   The DOM lets you navigate to a subsection of a JSON payload and deserialize a single value, a custom type, or an array. For information about the `JsonNode` DOM in .NET 6, see [Deserialize subsections of a JSON payload](system-text-json-use-dom-utf8jsonreader-utf8jsonwriter.md?pivots=dotnet-6-0#deserialize-subsections-of-a-json-payload). For information about the `JsonDocument` DOM, see [How to search a JsonDocument and JsonElement for sub-elements](system-text-json-migrate-from-newtonsoft-how-to.md#how-to-search-a-jsondocument-and-jsonelement-for-sub-elements).
```

Comment:
Any reason the `JsonNode` and `JsonDocument` aren't xrefs?

Reply:
I was thinking the API ref wouldn't be as helpful as the other links I'm providing, but on second thought it can't hurt to make them xref links.

```suggestion
>   The DOM lets you navigate to a subsection of a JSON payload and deserialize a single value, a custom type, or an array. For information about the <xref:System.Text.Json.Nodes.JsonNode> DOM in .NET 6, see [Deserialize subsections of a JSON payload](system-text-json-use-dom-utf8jsonreader-utf8jsonwriter.md?pivots=dotnet-6-0#deserialize-subsections-of-a-json-payload). For information about the <xref:System.Text.Json.JsonDocument> DOM, see [How to search a JsonDocument and JsonElement for sub-elements](system-text-json-migrate-from-newtonsoft-how-to.md#how-to-search-a-jsondocument-and-jsonelement-for-sub-elements).
```